### PR TITLE
Check in of the full MySQL based clinical tutorial

### PR DESCRIPTION
**Pull request by henryhbk**
This is the clinical tutorial based on MySQL for Connect-Client. merging into the main project

### CODE REVIEW

File: client-tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            `mysql -u<your_user_name> -p kafkaFhirDemoDb < 1.sql`
+note there are no spaces between `-u` and your username. The -p does support password entry in the command but that is a huge security risk as you password is now in your shell history (it will even chastise you). You can of course rename the database from *kafkaFhirDemoDb* but you will need to edit *lfh_demodb.sql* and the python code (specifically in *DatabaseUtil.createConnectionPool()*
+
+Note: of course you will need to edit the database url *DatabaseUtil.createConnectionPool()* to match your environment, including username/password and IP address. You of course need to specify in your mysql server that the user you select can access from an outside host if you are not running all this on localhost.
+
+## Running
+Assuming you got the docker stuff above launched, you now need to run EntryPoint.py to start the application. It will launch a desktop application called "LfhTutorial" with the strange Kivy GUI (see below)
+
```
Comment (ccorley) [inline client-tutorial/README.md:94]: At this point, I'd expect to see a code block that shows the command needed to `run EntryPoint.py to start the application`.  Also, it helps to indicate where the user should be (top-level directory, etc) when they run the command.

Comment (henryhbk) [reply client-tutorial/README.md:94]: Wouldn't the convention to run a class be to be in the directory containing the file?

File: client-tutorial/administrative_data.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# The administrative data part of the tutorial
+## Henry Feldman MD, FACP, FHM, FAMIA
+## CMO for Development
+
```
Comment (ccorley) [inline client-tutorial/administrative_data.md:4]: I wouldn't include your name and title in the open source doc, but your name would be in github as the contributor.

File: client-tutorial/administrative_data.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+### this part of the tutorial is to use the administrative (billing) data in the tutorial database.
+In the database we include several tables and classes in python that represent these adminiatrative data elements. The administrative data is arranged slightly hierarchically, similar to the order of the clinical workflow.
+
+the code for this section is in [billing.py] (billing.py) which you should run 
+
+All the administrative type data is handled by [ADTDao.py] (ADTDao.py). In this case the *Admission* class is the first item to select. You should fetch 
+the admissions for the patient (the admissions are mapped via the *subject_id* field). The admissions have an id that indexes into the other
```
Comment (ccorley) [inline client-tutorial/administrative_data.md:11]: I think the links will render properly if the space between [billing.py] (billing.py) and [ADTDao.py] (ADTDao.py) is removed.

**Review by ccorley — CHANGES_REQUESTED** [review]
Looks good!  I had a few comments, plus we shouldn't check in .DS_Store changes.  You can delete those locally and then check in those updates.  You can also optionally add a .gitignore that filters that file.

**Review by ccorley — APPROVED** [review]

**Pull request merged**